### PR TITLE
Fix grammar and construction in 2.1.10

### PR DESCRIPTION
**Pull request by galacemiguel**
There were various grammatical and construction-related errors in the [Window sizes and scrolling article](https://javascript.info/size-and-scroll-window). I did my best to correct them.

Please let me know if I made a mistake myself! Thank you!

### CODE REVIEW

File: 2-ui/1-document/10-size-and-scroll-window/article.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 How do we find the width and height of the browser window? How do we get the full width and height of the document, including the scrolled out part? How do we scroll the page using JavaScript?
 
-For most such requests, we can use the root document element `document.documentElement`, that corresponds to the `<html>` tag. But there are additional methods and peculiarities important enough to consider.
+For these, we can use the root document element `document.documentElement`, which corresponds to the `<html>` tag. But there are additional methods and peculiarities to consider.
```
Comment (paroche) [inline 2-ui/1-document/10-size-and-scroll-window/article.md]: I would leave this as is, or maybe replace "most such requests," with "for this type of information" (probably a little better w/out comma). And I prefer "that" over "which".

File: 2-ui/1-document/10-size-and-scroll-window/article.md [hunk 2]
```diff
@@ -16,33 +16,33 @@ For instance, this button shows the height of your window:
...
+Browsers also support properties like `window.innerWidth/innerHeight`. They look like what we want, so why not to use them instead?
 
-If there exists a scrollbar, and it occupies some space, `clientWidth/clientHeight` provide the width/height without it (subtract it). In other words, they return width/height of the visible part of the document, available for the content.
+If there exists a scrollbar, and it occupies some space, `clientWidth/clientHeight` provide the width/height without it (subtract it). In other words, they return the width/height of the visible part of the document, available for the content.
 
-...And `window.innerWidth/innerHeight` include the scrollbar.
+`window.innerWidth/innerHeight` includes the scrollbar.
 
-If there's a scrollbar, and it occupies some space, then these two lines show different values:
+If there's a scrollbar, and it occupies some space, then these two lines return different values:
```
Comment (paroche) [inline 2-ui/1-document/10-size-and-scroll-window/article.md]: I would stick with "show", especially since "return" has special meaning.

File: 2-ui/1-document/10-size-and-scroll-window/article.md [hunk 3]
```diff
@@ -16,33 +16,33 @@ For instance, this button shows the height of your window:
...
 ```js run
 alert( window.innerWidth ); // full window width
 alert( document.documentElement.clientWidth ); // window width minus the scrollbar
 ```
 
-In most cases we need the *available* window width: to draw or position something. That is: inside scrollbars if there are any. So we should use `documentElement.clientHeight/Width`.
+In most cases, we need the *available* window width—to draw or position something within scrollbars if there are any. So we should use `documentElement.clientHeight/clientWidth`.
```
Comment (paroche) [inline 2-ui/1-document/10-size-and-scroll-window/article.md]: Suggest: "In most cases we need the *available* window width, in order to draw or position something within the scrollbars (if there are any). So we should use `documentElement.clientHeight/clientWidth`."

File: 2-ui/1-document/10-size-and-scroll-window/article.md [hunk 4]
```diff
@@ -16,33 +16,33 @@ For instance, this button shows the height of your window:
...
 ````
 
 ```warn header="`DOCTYPE` is important"
-Please note: top-level geometry properties may work a little bit differently when there's no `<!DOCTYPE HTML>` in HTML. Odd things are possible.
+Please note: top-level geometry properties may work a little bit differently when there's no `<!DOCTYPE HTML>` in HTML.
```
Comment (paroche) [inline 2-ui/1-document/10-size-and-scroll-window/article.md]: I like the chatty little addition. Would leave it in.

File: 2-ui/1-document/10-size-and-scroll-window/article.md [hunk 5]
```diff
@@ -56,13 +56,13 @@ let scrollHeight = Math.max(
 alert('Full document height, with scrolled out part: ' + scrollHeight);
 ```
 
-Why so? Better don't ask. These inconsistencies come from ancient times, not a "smart" logic.
+Why so? Well, the gist of it is these inconsistencies come from ancient times.
```
Comment (paroche) [inline 2-ui/1-document/10-size-and-scroll-window/article.md]: Again, would leave as is. Humanizing. Even if 'not a "smart" logic' is a bit unusual, grammatically.

Comment (galacemiguel) [reply 2-ui/1-document/10-size-and-scroll-window/article.md]: It's too awkward I think. I found it to be inconsistent with the rest of the writing style in the article, especially when read top-down. Maybe we can rephrase it to be more idiomatic?

```suggestion
Why, you might ask? These inconsistencies come from ancient times, not bad design.
```

File: 2-ui/1-document/10-size-and-scroll-window/article.md (hunk 2 above)
Comment (galacemiguel) [reply 2-ui/1-document/10-size-and-scroll-window/article.md]: Good point. Will bring it back.

Comment (galacemiguel) [reply 2-ui/1-document/10-size-and-scroll-window/article.md]:
```suggestion
If there's a scrollbar, and it occupies some space, then these two lines show different values:
```

File: 2-ui/1-document/10-size-and-scroll-window/article.md (hunk 1 above)
Comment (paroche) [reply 2-ui/1-document/10-size-and-scroll-window/article.md]: Breaking the "that" clause out into a separate sentence doesn't seem as good to me as way it is in the original. Still favor my original suggestion.

File: 2-ui/1-document/10-size-and-scroll-window/article.md (hunk 3 above)
Comment (paroche) [reply 2-ui/1-document/10-size-and-scroll-window/article.md]: Could go with or without the comma there. No strong preference.

File: 2-ui/1-document/10-size-and-scroll-window/article.md (hunk 5 above)
Comment (paroche) [reply 2-ui/1-document/10-size-and-scroll-window/article.md]: Either way. I still prefer the original, though.

File: 2-ui/1-document/10-size-and-scroll-window/article.md (hunk 4 above)
Comment (paroche) [reply 2-ui/1-document/10-size-and-scroll-window/article.md]: Doesn't bother me the way it was, and the statement that "JavaScript can be quite finicky" seems to go beyond the scope of what is being discussed.

**Comment by iliakan** [comment]
/me stands with @paroche on the review comments, please update the PR :)

**Comment by iliakan** [comment]
@galacemiguel Could you please fix the PR according to the suggestions?

**Comment by iliakan** [comment]
ping @galacemiguel 😮

**Comment by galacemiguel** [comment]
> ping @galacemiguel 😮

Sorry, this one slipped my mind. 😅 

Getting to it now! @iliakan

File: 2-ui/1-document/10-size-and-scroll-window/article.md (hunk 1 above)
Comment (galacemiguel) [reply 2-ui/1-document/10-size-and-scroll-window/article.md:5]:
```suggestion
For this type of information, we can use the root document element `document.documentElement`, that corresponds to the `<html>` tag. But there are additional methods and peculiarities to consider.
```

File: 2-ui/1-document/10-size-and-scroll-window/article.md (hunk 4 above)
Comment (galacemiguel) [reply 2-ui/1-document/10-size-and-scroll-window/article.md:36]:
```suggestion
Please note: top-level geometry properties may work a little bit differently when there's no `<!DOCTYPE HTML>` in HTML. Odd things are possible.
```

File: 2-ui/1-document/10-size-and-scroll-window/article.md (hunk 5 above)
Comment (galacemiguel) [reply 2-ui/1-document/10-size-and-scroll-window/article.md:59]:
```suggestion
Why so? Better don't ask. These inconsistencies come from ancient times, not a "smart" logic.
```

**Comment by galacemiguel** [comment]
Hi, @iliakan! I've committed https://github.com/javascript-tutorial/en.javascript.info/pull/1672/commits/d3ad4c3d82db101b51da6f41de94aa351db2abfe to revert and apply some changes based on the suggestions.

Let me know if there's anything I missed.

**Comment by iliakan** [comment]
Wow! So I've got to review it again! ;)

**Comment by iliakan** [comment]
Great, thanks!
